package.json

```diff
@@ -32,6 +32,8 @@
     "@types/jest": "^25.2.1",
     "@types/react": "^16.9.34",
     "@types/react-dom": "^16.9.6",
+    "@types/react-modal": "^3.10.5",
```

Comment:
I think you are no longer using react-modal, zo this can also be removed from the package.json

Reply:
Yes, forgot this one